package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wt-cli",
-  "version": "11.0.0",
+  "version": "11.0.2",
```

Comment:
why did `package-lock.json` change, but not `package.json`?

Reply:
It looks like `package-lock.json` had not been updated with the last (two?) releases and when I ran `npm i` after pulling master, `npm` corrected this mismatch.